include/wx/private/smartrowheightcache.h

```diff
@@ -0,0 +1,142 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/private/smartrowheightcache.h
+// Purpose:     Smart height cache of rows in a dataview
+// Author:      Jens Goepfert
+// Created:     2018-03-06
+// Copyright:   (c) 2018 Vadim Zeitlin <vadim@wxwidgets.org>
```

Comment:
Not really. Please assign copyright to "wxWidgets team" if you don't care about it.

Reply:
i updated this